aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -20,8 +20,8 @@ spec aptos_framework::staking_contract {
 
     /// Staking_contract exists the stacker/operator pair.
     spec staking_contract_amounts(staker: address, operator: address): (u64, u64, u64) {
-        // TODO: set because of timeout (property proved)
-        pragma verify_duration_estimate = 1000;
+        // TODO: set because of timeout
```

Comment:
About those TODOs as it apparently is not properly proved? Here and elsewhere

Reply:
MoveBit claimed that these properties could be proved with longer verification time but I found that they will timeout when being verified along with other specs. So I changed the comments and will investigate them more.

Reply:
But the comment is still saying here and there that timeout is the problem.

Reply:
Yes, the spec itself is not problematic but timeout is the issue. They cannot be proved even if the verification time is set to 1000 so I changed the comment.